lib/jekyll-paginate-v2/generator/paginationModel.rb

```diff
@@ -251,17 +251,23 @@ def paginate(template, config, site_title, all_posts, all_tags, all_categories,
           paginated_page_url = config['permalink']
           first_index_page_url = ""
           if template.data['permalink']
-            first_index_page_url = template.data['permalink']
+            first_index_page_url = template.data['permalink']+'/'
```

Comment:
As far as I tested, the concatenation is done by File.join, which takes care of the double slashes. That could still cause bugs in the future, so i'll fix it anyways.